dts/arm/st/l0/stm32l0.dtsi

```diff
@@ -180,6 +180,15 @@
 			#io-channel-cells = <1>;
 		};
 
+		dac1: dac@40007400 {
+			compatible = "st,stm32-dac";
+			reg = <0x40007400 0x400>;
+			clocks = <&rcc STM32_CLOCK_BUS_APB1 0x20000000>;
+			status = "disabled";
+			label = "DAC_1";
+			#io-channel-cells = <1>;
```

Comment:
Have you checked https://www.kernel.org/doc/Documentation/devicetree/bindings/iio/dac/st%2Cstm32-dac.txt ?
Ideally we should match this description.

Reply:
I have checked it, but not sure if we can make it compatible. As far as I understand, we don't have a phandle for vref available, for example. Also the Zephyr ADC driver seems quite different from the Linux one. I took the ADC as a basis.